dexter/build.gradle

```diff
@@ -2,12 +2,12 @@ apply plugin: 'com.android.library'
 apply from: 'https://raw.github.com/chrisbanes/gradle-mvn-push/master/gradle-mvn-push.gradle'
 
 android {
-  compileSdkVersion 25
-  buildToolsVersion '25.0.0'
+  compileSdkVersion 27
+  buildToolsVersion '27.0.3'
 
   defaultConfig {
-    minSdkVersion 10
-    targetSdkVersion 25
+    minSdkVersion 14
```

Comment:
Hi @rid-hrant I'm taking a look at this PR and I was wondering why you've increased the ``minSdkVersion``. This change is huge right now. As we can review in these pages, the rest of the updates are correct and we are using the latest version released:

https://developer.android.com/studio/releases/gradle-plugin
https://developer.android.com/studio/releases/build-tools
https://developer.android.com/topic/libraries/support-library/revisions

But we should review this minSdkVersion carefully right @Serchinastico? I'm ok if we update the min SDK but we should keep it in mind and at least add this information to the release notes.

Reply:
Hi my friend :smile: . I try to use dexter in my updated gradle version poroject, but it has conflict with support libs, so I decided to contribute and update libs and gradle version.
Min sdk version 10 has conflict with new support libs 27 version, so I had to update version to 14, 14 is android stidio's recomendid version.

Thanks for answering.
Sorry for bad english